worker/buildbot_worker/runprocess.py

```diff
@@ -638,10 +639,10 @@ def _spawnProcess(
                     processProtocol, executable, args, env, path, usePTY=usePTY
                 )
             pHandle = win32api.OpenProcess(win32con.PROCESS_ALL_ACCESS, False, int(process.pid))
-
-            # use JobObject to group subprocesses
-            self.job_object = self._create_job_object()
-            win32job.AssignProcessToJobObject(self.job_object, pHandle)
+            if win32process.GetExitCodeProcess(pHandle) == win32con.STILL_ACTIVE:
```

Comment:
might be worth it to look into the `CREATE_SUSPENDED` flag to handle cases where a command will directly spawn a process and exit (although this is good for a quick fix).

Reply:
is it even possible with reactor.spawnProcess to use flags? If no, it can be very challenging rewrite this part to win32process.

Reply:
If [this](https://github.com/twisted/twisted/blob/trunk/src/twisted/internet/_dumbwin32proc.py#L191-L194) is the implementation used by twisted for process on Windows (not sure, only did a quick search), it doesn't look like it.
Probably not event possible to subclass it since everything is done in the init...

Reply:
I also did not find anything. So is this sufficient fix or need to be rework some other way?